generator/internal/genclient/model.go

```diff
@@ -230,6 +230,8 @@ type Field struct {
 	JSONName string
 	// Optional indicates that the field is marked as optional in proto3.
 	Optional bool
+	/// True if the field is a repeated field.
```

Comment:
Nit: Go documentation usually start with the noun, so `Repeated is...` (see https://tip.golang.org/doc/comment)